packages/postcss-colormin/src/lib/color.js

```diff
@@ -0,0 +1,67 @@
+import { colord, extend, getFormat } from 'colord';
+import namesPlugin from 'colord/plugins/names';
+import getShortestString from './getShortestString';
+
+let minifierPlugin = (Colord) => {
+  /**
+   * Shortens a color to 3 or 4 digit hexadecimal string if it's possible.
+   * Returns the original (6 or 8 digit) hex if the it can't be shortened.
+   */
+  Colord.prototype.toShortHex = function ({ formatAlpha }) {
```

Comment:
Have you thought of making this plugin part of the colord package instead so it could also be used outside of cssnano (maybe more people would try the minification so it would become better over time, although it could make life harder for cssnano as it would lose control of some of the minification logic, but )

Reply:
Yes, I have thought) But, since we're having some browser support conditions, I didn't wonder a good way to create a good plugin that can be used here.

If I create the plugin, I'll definitely come back and simplify your code =)